docs/markdown/snippets/dataonly-pkgconfig-default-install-path.md

```diff
@@ -0,0 +1,4 @@
+## `dataonly` Pkgconfig Default Install Path
+
+The default install path for `dataonly` pkgconfig files has changed from
+`${prefix}/${libdir}/pkgconfig` to `${prefix}/${datadir}/pkgconfig`.
```

Comment:
Nit: it's just libdir/datadir, and prefix is inferred for relative paths -- which actually changed just recently to allow absolute paths outside of prefix.

We don't need to spell out the prefix here. Also I'm not sure we need to join it to the datadir in the code either... Relative install_dir paths are a first class feature.

Reply:
Ok sounds good. I will make the changes in both places.